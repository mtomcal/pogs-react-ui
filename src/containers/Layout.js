import React, {PropTypes, Component} from 'react';
import Navbar from './Navbar';

class Layout extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }
  render() {
    let styles = {};

    return (
      <div>
          <Navbar />
          <div className="container">
              <div className="row">
                  <div className="col-sm-10 col-sm-pull-1 col-sm-push-1">
                      {this.props.children}
                  </div>
              </div>
          </div>
      </div>
    );
  }
}

Layout.propTypes = {
    children: PropTypes.element
};

export default Layout;
