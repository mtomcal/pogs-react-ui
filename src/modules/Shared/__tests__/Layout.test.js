import React from 'react';
jest.mock('react-router-dom');
jest.mock('react-router');
import Layout from '../Layout';
import renderer from 'react-test-renderer';

test('renders with one element', () => {
  const component = renderer.create(
    <Layout>
      <h1>Stuff</h1>
    </Layout>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

