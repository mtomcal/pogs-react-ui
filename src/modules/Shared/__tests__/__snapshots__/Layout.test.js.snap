// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with one element 1`] = `
<div>
  <div
    className="z-depth-1"
  >
    <nav
      className="navbar navbar-default"
    >
      <div
        className="container-fluid"
      >
        <div
          className="row col-sm-12"
        >
          <div
            className="navbar-header"
          />
        </div>
      </div>
    </nav>
  </div>
  <div
    className="container-fluid"
  >
    <h1>
      Stuff
    </h1>
  </div>
</div>
`;
